storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6772,26 +6772,26 @@ static void xtrabackup_prepare_func(int argc, char **argv) {
   if (opt_transition_key) {
     if (!xb_tablespace_keys_load(xtrabackup_incremental, opt_transition_key,
                                  strlen(opt_transition_key))) {
-      msg("xtrabackup: Error: failed to load tablespace "
-          "keys\n");
+      xb::error() << "failed to load tablespace "
+                     "keys";
```

Comment:
check formatting, this should fit same line as above.

Reply:
yeah, if we split, clang-format is not able to join them. I will do it manually.